assignments/function-conversion.js

```diff
@@ -1,30 +1,30 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
-// console.log("Function was invoked!");
-// };
-// myFunction();
+let myFunction =  () => {
+console.log("Function was invoked!");
+};
+myFunction();
 
-// let anotherFunction = function (param) {
-//   return param;
-// };
-// anotherFunction("Example");
+let anotherFunction = (param) => {
+  return param;
+};
+anotherFunction("Example");
 
-// let add = function (param1, param2) {
-//   return param1 + param2;
-// };
-// add(1,2);
+let add = (param1, param2) => {
+  return param1 + param2;
+};
+add(1,2);
 
-// let subtract = function (param1, param2) {
-//   return param1 - param2;
-// };
-// subtract(1,2);
+let subtract = (param1, param2) => {
+  return param1 - param2;
+};
+subtract(1,2);
 
 
 // Stretch
 
-// exampleArray = [1,2,3,4];
-// const triple = exampleArray.map(function (num) {
-//   return num * 3;
-// });
-// console.log(triple);
\ No newline at end of file
+exampleArray = [1,2,3,4];
+const triple = exampleArray.map( (num) => {
+  return num * 3;
+});
```

Comment:
You can even simplify these further since they only require one return statement:
```
const triple = exampleArray.map(num => num*3);
```